### ISSUE TITLE
Invalid position given by 'W033'

### ISSUE THREAD

**Issue by BenCommet** [AtomLinter/linter-jshint#480]
JSHint returned a point that did not exist in the document being edited.
Rule: `W033`
Requested point: 99:50

var szGame = szGame || {};

Debug information:
```
Atom version: 1.19.0
linter-jshint version: v3.1.5
JSHint version: jshint v2.9.5
Hours since last Atom restart: 1.1
Platform: darwin
Current file's scopes: [
  "source.js"
]
linter-jshint configuration: {
  "executablePath": "",
  "lintInlineJavaScript": false,
  "disableWhenNoJshintrcFileInPath": false,
  "scopes": [
    "source.js",
    "source.js-semantic"
  ],
  "jshintFileName": ".jshintrc",
  "jshintignoreFilename": ".jshintignore"
}
```

**Comment by Arcanemagus** [comment]
I get no warning messages from a file containing:

```js
var szGame = szGame || {};
```

I tested with these files: [linter-jshint_GH480.zip](https://github.com/AtomLinter/linter-jshint/files/1216228/linter-jshint_GH480.zip)

Are you sure that code alone is giving you this message? Are you seeing it with the files in that zip?

**Comment by Arcanemagus** [comment]
CLosing due to a lack of response and inability to reproduce with the given code.
